members/README.md

```diff
@@ -91,3 +105,30 @@ Returns all inactive/idle members in the system.
 - **Error Response:**
   - **Code:** 500
     - **Content:** `{ 'statusCode': 500, 'error': 'Internal Server Error', 'message': 'Something went wrong. Please contact admin' }`
+
+## POST /members/intro/:username
+Creates new request for member introduction 
+
+- **Params**\
+    _Required_: `username=[string]`
+- **Query**\
+    None
+- **Body**\
+    `{<recruiter_object>}`
+- **Headers**\
+    None
+- **Cookie**\
+    None
+- **Success Response:**
+    - **Code:** 200
+        - **Content:** 
+        ```
+        {
+            message: 'Request Submission Successful!!'
+            id: <new unique submission id>
```

Comment:
Do we need to send `id` back? I guess the message should be sufficient
Also, it'd make more sense if the response code is `201`

Reply:
It's just like a coupon for the recruiter so that later on one request can be differentiated from another request.

Reply:
What I'm saying is that do we need to return the `id` in response?

Reply:
Suppose, think of a situation, where they approach 5 Candidates for a particular role and later on they contact RDS for further queries regarding a particular candidate or a few of the selected one's. How is he going to approach us and how are we going to separate or distinguish particular requests without a submission id? It is like an order number for an item we order from amazon so that if we have a query we can reference our order to customer care via our order ID. That's what I'm saying. We both(RDS and the recruiter) must have a submission ID for future references.                              And maybe there is some other POV you trying to convey that I cannot perceive. Please provide your direction so that I can approach it, taking care of your way of perception.